tests/testthat/test-text.R

```diff
@@ -18,6 +18,17 @@ test_that("renderPrint and renderText behavior is correct", {
                'foo')
   expect_equal(isolate(renderText({ invisible("foo") })()),
                'foo')
+  expect_equal(isolate(renderText({ c("hello", "world") })()),
+               'hello world')
+
+  expect_equal(isolate(renderText({ c("hello", "world") }, sep=", ")()),
```

Comment:
I wonder if it’s worth suppressing the reactive context error by default inside of tests so that you don’t have to put isolate everywhere.